### PR TITLE
[GH Actions] Enable lock action

### PR DESCRIPTION
**Pull request by mollyIV**
### Description

We are working on a bot's code migration to GitHub Actions 👉 fastlane/github-actions#3.
This PR is about enabling locking closed, inactive issues and pull requests workflow.

Enabling this workflow will not conflict with an issue-bot processing flow. 

### Testing Steps

The workflow file used for testing the action 👇

```yml
name: Locking issues and pull requests
on:
  schedule:
  - cron: "0 0 * * *"

jobs:
  lock:
    runs-on: ubuntu-latest
    steps:
    - uses: fastlane/github-actions/lock@latest
      with:
        repo-token: ${{ secrets.GITHUB_TOKEN }}
        days-before-lock: 1
```

image

image

> BOT_GITHUB_TOKEN is set to my personal access token, that's why you see my personal avatar and "Thanks for your contribution."  comment in a pull request comes from a different GH Action.

### CODE REVIEW

File: lock.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - cron: "0 0 * * *"
+
+jobs:
+  lock:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: fastlane/github-actions/lock@latest
+      with:
+        repo-token: ${{ secrets.BOT_GITHUB_TOKEN }}
+        days-before-lock: 60
```
Comment (mollyIV) [inline lock.yml:14]: The time before locking is taken from an original bot's implementation:

https://github.com/fastlane/issue-bot/blob/457348717d99e5ffde34ca1619e7253ed51ec172/bot.rb#L286-L292

**Comment by joshdholtz** [comment]
@mollyIV Heyyy 😊 I'm slightly torn about this one, TBH 🤔 I've seen quite a bit of backlash about locking issues and PR lately. Closing seems totally okay but I'm not sure if we should auto lock since it prohibits communication and than can frustrate some members of the community. Do you have any thoughts or experienced with locked issues in other communities?

**Comment by mollyIV** [comment]
Hello Josh 👋 Actually the current implementation of our active bot is locking the issues:

https://github.com/fastlane/issue-bot/blob/457348717d99e5ffde34ca1619e7253ed51ec172/bot.rb#L139

Thought that we want to keep that functionality - that's why we've created a [lock GH Action](https://github.com/fastlane/github-actions/tree/master/lock) 😊  

> Do you have any thoughts or experienced with locked issues in other communities?

TBH I am okay with locking the closed issues after a bit longer time, like 2 months or something. Opening new issues might be more appreciated than digging deep into old, already closed conversation, wdyt? 🤔

**Review by janpio — COMMENTED** [review]
Moving more logic into the GH Actions makes sense 👍

**Review by janpio — APPROVED** [review]
Final confirmation before merging @mollyIV: This just recreates already existing behavior?

**Comment by mollyIV** [comment]
According to [the source code of an existing bot](https://github.com/fastlane/issue-bot/blob/457348717d99e5ffde34ca1619e7253ed51ec172/bot.rb#L287-L304) - yes, it does. Here you can find an example of locked issue via the existing bot:
* https://github.com/fastlane/fastlane/issues/15747

**Pull request merged**